uopz.h

```diff
@@ -51,6 +51,10 @@ ZEND_END_MODULE_GLOBALS(uopz)
 #define uopz_exception(message, ...) zend_throw_exception_ex\
 	(spl_ce_RuntimeException, 0, message, ##__VA_ARGS__)
 
+#if PHP_VERSION_ID < 80100
+#define ZEND_VERIFY_NEVER_TYPE         201
+#endif
```

Comment:
Hmm, instead of using a hard-coded value here, can't we conditionally do the changes in src/handlers.c.

Reply:
Sure, updated